docs/src/pages/guides/probes.md

```diff
@@ -75,6 +75,23 @@ Details of the field are given in the table below.
 | ping (optional)              | (boolean), If set true then send a PING to the specified url instead.                                                                                                                                                                                                                                                                                     |
 | allowUnauthorized (optional) | (boolean), If set to true, will make https agent to not check for ssl certificate validity                                                                                                                                                                                                                                                                |
 
+### Mariadb Request
+
+To monitor connectivity and health of your MariaDB database, monika provides a MariaDB database probe, set it up like below:
```

Comment:
To monitor the connectivity and the health of your MariaDB/MySQL database, you can use the following monika configuration